analyzers/src/SonarAnalyzer.Common/Extensions/SyntaxTokenExtensions.cs

```diff
@@ -24,4 +24,7 @@ public static class SyntaxTokenExtensions
 {
     public static int Line(this SyntaxToken token) =>
         token.GetLocation().StartLine();
+
+    public static SecondaryLocation ToSecondaryLocation(this SyntaxToken token, string message = null) =>
```

Comment:
Consider adding overloads for `this IEnumerable<SyntaxToken> tokens` for all other extension methods. The pattern seems common enough.

Reply:
I took a look and I haven't seen many cases where this would be useful.